docs_guidelines/conf.py

```diff
@@ -26,12 +26,11 @@
 
 html_theme = 'qiskit_sphinx_theme'
 
-# -- Options for Autosummary and Autodoc -------------------------------------
+# -- Intersphinx configuration ------------------------------------------------
 
-# Note that setting autodoc defaults here may not have as much of an effect as you may expect; any
-# documentation created by autosummary uses a template file (in autosummary in the templates path),
-# which likely overrides the autodoc defaults.
-
-autosummary_generate = True
-autosummary_generate_overwrite = False
-autoclass_content = "both"
+intersphinx_mapping = {
+    "python": ("https://docs.python.org/3", None),
+    "numpy": ("https://numpy.org/doc/stable/", None),
+    "scipy": ("https://docs.scipy.org/doc/scipy/", None),
```

Comment:
Do we need to link to `python`, `numpy` and `scipy`? Can you remove them if they are not used?

Reply:
There's no need. I'll remove them